libdrgn/python/type.c

```diff
@@ -568,6 +611,8 @@ static PyObject *DrgnType_repr(DrgnType *self)
 		goto out_repr_leave;
 	if (append_member(parts, self, &first, is_variadic) == -1)
 		goto out_repr_leave;
+	if (append_member(parts, self, &first, template_parameters) == -1)
+		goto out_repr_leave;
```

Comment:
Could you make this conditional on the type having template parameters? That'll make `repr()` less noisy for the common case of no parameters.